DQM/L1TMonitor/src/L1TdeStage2CPPF.cc

```diff
@@ -0,0 +1,464 @@
+#include <string>
+
+#include "DQM/L1TMonitor/interface/L1TdeStage2CPPF.h"
```

Comment:
If these are plugins, please:
- Move them to the \plugin area
- Unify .h and .cc into a unique .cc in \plugin